Removing the Webspace mention from the documentation

Chibionos opened the pull request:
Webspace mentioned in the documentation is confusing the customers as it is an internal implementation detail and causes a lot of issues as we do not surface it anywhere on our portal. Nowhere in our documentation, we should be mentioning the word WebSpace.

azurecla commented:
Hi __@sachouks__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (sachouks). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

Chibionos committed:
Removing the Webspace mention from the documentation

Webspace mentioned in the documentation is confusing the customers as it is an internal implementation detail and causes a lot of issues as we do not surface it anywhere on our portal. Nowhere in our documentation, we should be mentioning the word WebSpace.
